lib/Target/TTNN/TTNNToFlatbuffer.cpp

```diff
@@ -290,6 +290,40 @@ createOp(FlatbufferObjectCache &cache, AllGatherOp op) {
                                                op.getDim(), op.getNumLinks());
 }
 
+::flatbuffers::Offset<::tt::target::ttnn::ClampOpParams>
+createEltwiseOpParams(FlatbufferObjectCache &cache, ClampOp op) {
+  auto min = op.getMin().convertToFloat();
+  auto max = op.getMax().convertToFloat();
+  return ::tt::target::ttnn::CreateClampOpParams(*cache.fbb, min, max);
+}
+
+template <typename EltwiseOp>
+::flatbuffers::Offset<::tt::target::ttnn::EltwiseOp>
+createNonDPSEltwiseOp(FlatbufferObjectCache &cache, EltwiseOp op) {
```

Comment:
Should we create an issue on the Metal side to support this op in the DPS style?

Reply:
I prefer to check-in the current implementation to enable brining up the models which require clamp op. 
I'll create an issue on the Metal side and keep track of the implementation.

Reply:
Issue created for tt-metal repo https://github.com/tenstorrent/tt-metal/issues/14855

Reply:
Thanks for creating this issue!